app/services/action_queue.rb

```diff
@@ -6,8 +6,9 @@ module ActionQueue
   module Enqueable
     extend ActiveSupport::Concern
 
-    def initialize(action)
+    def initialize(action, mailing_id)
```

Comment:
Does the action have the `akid`? If so, we could get the `mailing_id` from there.

Reply:
It does not :/ unless it maybe has it in the form_data jsonb field, but I'm not sure I'd rely it on always being present there